x-pack/plugins/apm/public/components/app/transaction_overview/index.tsx

```diff
@@ -99,12 +99,6 @@ export function TransactionOverview({ serviceName }: TransactionOverviewProps) {
     [serviceName, transactionType]
   );
 
-  // TODO: improve urlParams typings.
-  // `serviceName` or `transactionType` will never be undefined here, and this check should not be needed
-  if (!serviceName || !transactionType) {
-    return null;
-  }
```

Comment:
I assume this was there for a reason? Are we linking to this page anywhere without a transaction type?

Reply:
yes, the service overview tab links to the transactions tab and it does not pass the transaction type with it.

Reply:
But shouldn't it pass the transaction type?

Reply:
well, it should let me add it and see what happens.

Reply:
> But shouldn't it pass the transaction type?

@dgieselaar if we have one we should, but like I said here https://github.com/elastic/kibana/pull/87298#discussion_r552457868, transaction type is undefined.

Reply:
Forget what I said 👆🏻, I'll do like we talked.